add Seth Vargo recap (#2982)
sourcegraph/about: blogposts/2021/dev-tool-time-seth-vargo.md

---

> title: Dev Tool Time with Seth Vargo - April 14, 2021

Reviewer: ```suggestion
title: Dev Tool Time with Seth Vargo: Productivity hacks and .gitconfig tips
```

Reviewer: I think the date of the episode is less important than giving readers an idea of what to expect from the post 😃

> title: Dev Tool Time with Seth Vargo: Productivity hacks and .gitconfig tips

Reviewer: ```suggestion
title: "Dev Tool Time with Seth Vargo: Productivity hacks and .gitconfig tips"
```

> description: On April 14, 2021, Dev Tool Time welcomed Seth Vargo as our first guest. Seth showed us his productivity optimized desk setup, talked about great window management and keyboard shortcuts. He shared his .gitconfig and much more.

Reviewer: Check out the recording of our first episode of Dev Tool Time, in which Google Cloud Engineer Seth Vargo shares his tips for a productivity-optimized desk setup, efficient window management, and keyboard shortcuts.

Reviewer: ```suggestion
description: Check out the recording of our first episode of Dev Tool Time, in which Google Cloud Engineer Seth Vargo shares his tips for a productivity-optimized desk setup, efficient window management, and keyboard shortcuts.
```

author: Vanesa Ortiz
authorUrl: https://twitter.com/vanesacodes
publishDate: 2021-04-28T10:00-07:00
tags: [blog]
slug: dev-tool-time-seth-vargo
heroImage: https://sourcegraphstatic.com/Twitter-Feed-Seth-Vargo-WS.png
socialImage: https://sourcegraphstatic.com/Twitter-Feed-Seth-Vargo-WS.png
published: true
---

> On April 14, 2021, Dev Tool Time welcomed [Seth Vargo](#seth-vargo) as our first guest. You can watch the live stream video on [YouTube Live](https://www.youtube.com/watch?v=sltjIaAwWac) or at the [bottom](#seth-vargo) of this blog post.

Reviewer: ```suggestion
Dev Tool Time is our bi-monthly virtual event series in which guest developers share their favorite dev tools and productivity hacks. We welcomed [Seth Vargo](#seth-vargo) as our first guest on April 14, 2021. You can watch the live stream video on [YouTube Live](https://www.youtube.com/watch?v=sltjIaAwWac) or at the [bottom](#seth-vargo) of this blog post.
```

Reviewer: Included a brief description of Dev Tool Time for people unfamiliar with it

Seth was generous in sharing his dev productivity secrets, including how to leverage keyboard shortcuts, a hack for setting up a new computer in record time, and his [.gitconfig](https://gist.github.com/sethvargo/d10a81f219f6469889269af2076b4d39) containing the [Git aliases](https://git-scm.com/book/en/v2/Git-Basics-Git-Aliases) you need to become a Git ninja.

He also talked about the importance of screen real estate and showed us his desk setup that’s optimized for productivity. Here is a summary of the setup and tools he mentioned:

## Seth’s desk setup for productivity

![Seth Vargo Desk Setup](https://sourcegraphstatic.com/SethVargo-DeskSetup.jpg)

![Seth Vargo Desk Setup](https://sourcegraphstatic.com/SethVargo-DeskSetup2.jpg)

> ### [LG 5k monitor](https://www.apple.com/shop/product/HMUB2LL/A/lg-ultrafine-5k-display)

Reviewer: ```suggestion
### [LG 5K monitor](https://www.apple.com/shop/product/HMUB2LL/A/lg-ultrafine-5k-display)
```

This monitor comes with a built-in USB-C hub to charge his MacBook at 96 watts, his Stream Deck and to connect his [Thunderbolt-based wired Ethernet](https://eshop.macsales.com/item/OWC/TB3ADP10GBE/).

### [Stream Deck](https://www.elgato.com/en/stream-deck)

Each Stream Deck button is assigned to run something, for example one button shows his CPU usage, another the CI status when he’s running a build, while another sends him straight to his GitHub notifications. He loves that it’s highly customizable, adapts to the current app, integrates with a ton of community-built apps, accelerates his workflow, and adds a little fun factor.

![Seth Vargo Stream Deck](https://sourcegraphstatic.com/SethVargo-StreamDeck.jpg)

> ### [Elgato 4k Cam Link](https://www.elgato.com/en/cam-link-4k)

Reviewer: ```suggestion
### [Elgato 4K Cam Link](https://www.elgato.com/en/cam-link-4k)
```
